### DISCUSSION TITLE
Why "Camera::setCursor()" method needed?

### DISCUSSION THREAD

**Discussion by itlancer** [airsdk/Adobe-Runtime-Support#1894] [Q&A] [answered]
Why "Camera::setCursor()" method needed?
It not documented in AIR reference but IDE offer such hint:
![image](https://user-images.githubusercontent.com/10899066/167311756-b05f16a3-e635-4861-b859-30c57cc2fbff.png)

**Comment by 2jfw** [comment]
Not sure what IDE you are using but the method is marked as ` [Inspectable(environment="none")]` - IntelliJ IDEA e.g. does _not_ offer the `setCursor` function. Tooltip in IntelliJ also states: "Instance member is not accessible".

**Reply by itlancer** [reply]
Such hint tooltip appears for me at VSCode and Adobe Animate.

**Comment by ajwfrost** [comment]
Interesting .. the method is also marked as "private" in the asdoc comment which I would have thought/hoped would mean it didn't get turned into the documentation!

It's actually just a stub function, doesn't do anything in the code now, it appears to be left over from when the Flash Player was used in other Adobe products with some custom functionalities (in this case, screen sharing...)

We can just delete this function without a problem (although getting it out of the documentation will take an extra step or two...)

**Reply by itlancer** [reply]
@ajwfrost 
Thank you for clarification!
Yes, I think this method should be deleted if it do nothing and doesn't needed anymore.

**Comment by itlancer** [comment, marked as answer]
This legacy method removed with AIR 50.0.0.1.
